test/allowlist_for_publicAPI.json

```diff
@@ -2404,6 +2404,8 @@
     "linear",
     "logsigmoid",
     "native_channel_shuffle",
+    "native_pixel_shuffle",
```

Comment:
please undo the changes in this file. This should be fixed by https://github.com/pytorch/pytorch/pull/75801

Reply:
@anjali411 thanks for reminding! This change is removed!